arc/cli/loading_interface.py

```
@@ -0,0 +1,329 @@
+
+
+class ConfigAnalysisLoader:
+    """Loading interface for agent configuration analysis."""
+    
+    def __init__(self, console: Console):
+        """Initialize with console for output."""
+        self.console = console
+        self.current_progress = None
+        
+    async def analyze_config_with_progress(self, config_path: str, 
+                                         parser, normalizer) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)